i18n/src/login/translation/defaultMessages/errors.json

```diff
@@ -11,6 +11,14 @@
     "id": "required",
     "defaultMessage": "Field cannot be empty"
   },
+  {
+    "id": "blank space not allowed",
```

Comment:
Let's not bother the user about spaces at the start or end of their input. Let's just take them out at the very beginning of validation and remove any communication back to the user about it.